.github/pull_request_template.md

```diff
@@ -21,6 +21,6 @@ How did you make sure this worked? How can a reviewer verify this?
 - [ ] Update the [release notes](../docs/release_notes.rst): reference the PR and related issues.
 - [ ] Ensure docs build, unit & integration tests, and test coverage pass locally with `make pytest-coverage` (otherwise the merge queue may reject your PR)
 - [ ] Review the PR yourself and call out any questions or issues you have
-- [ ] For minor ETL changes or data additions, once `make pytest-coverage` passes, make sure you have a fresh full PUDL DB downloaded locally, materialize new/changed assets and all their downstream assets and [run relevant data validation tests] (https://catalystcoop-pudl.readthedocs.io/en/latest/dev/testing.html#data-validation) using `pytest` and `--live-dbs`.
-- [ ] For significant ETL, data coverage or analysis changes, once `make pytest-coverage` passes, ensure the full ETL runs locally and [run all data validation tests] (https://catalystcoop-pudl.readthedocs.io/en/latest/dev/testing.html#data-validation) using `make pytest-validate` (together, expect a ~10 hour run). If you can't run this locally , run the `build-deploy-pudl` GitHub Action (or ask someone else with permissions to do so on your behalf). Then, check the `#pudl-deployments` Slack channel or [builds.catalyst.coop] (builds.catalyst.coop) to find the logs for the validation tests and ensure all tests passed.
+- [ ] For minor ETL changes or data additions, once `make pytest-coverage` passes, make sure you have a fresh full PUDL DB downloaded locally, materialize new/changed assets and all their downstream assets and [run relevant data validation tests](https://catalystcoop-pudl.readthedocs.io/en/latest/dev/testing.html#data-validation) using `pytest` and `--live-dbs`.
+- [ ] For significant ETL, data coverage or analysis changes, once `make pytest-coverage` passes, ensure the full ETL runs locally and [run data validation tests](https://catalystcoop-pudl.readthedocs.io/en/latest/dev/testing.html#data-validation) using `make pytest-validate` (a ~10 hour run). If you can't run this locally, run the `build-deploy-pudl` GitHub Action (or ask someone with permissions to). Then, check the logs on the `#pudl-deployments` Slack channel or [builds.catalyst.coop](builds.catalyst.coop).
```

Comment:
We can't provide a real link to the build outputs. E.g. this link is broken in multiple ways: [builds.catalyst.coop](builds.catalyst.coop). We can explicitly say `gs://builds.catalyst.coop` and not create a link though.